modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -60,6 +67,14 @@ public void testExportImportObjectDefinition() throws Exception {
 		testExportImport(
 			"test-object-definition.json", "test-object-definition.json", null,
 			"TestObjectDefinition");
+
+		JSONObject actualJSONObject = _baseObjectDefinitionJSONObject;
+		JSONObject expectedJSONObject = _baseObjectDefinitionJSONObject;
+
+		testExportImport(
+			"actualFileName.json", JSONUtil.toString(actualJSONObject),
+			JSONUtil.toString(expectedJSONObject), null,
```

Comment:
Change those to `_baseObjectDefinitionJSONObject.toString()`. Dunno if its better to create a variable for it and call it twice here. Your call